src/accessibilityStatement/AccessibilityStatement.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import styles from './AccessibilityStatement.module.css';
+import Header from '../common/header/Header';
+import Footer from '../common/footer/Footer';
+import AccessibilityStatementFi from './AccessibilityStatementFi';
+import AccessibilityStatementSv from './AccessibilityStatementSv';
+import AccessibilityStatementEn from './AccessibilityStatementEn';
+
+function AccessibilityStatement() {
+  const { i18n } = useTranslation();
+  const selectStatement = () => {
+    const lang = i18n.languages[0];
```

Comment:
Might this language value also sometimes contain the locale information? If I remember correctly you configured locales with both youth-membership and profile.

Reply:
You are totally right, nice catch 👍